rll-examples/install-packages.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+# ---
+# RightScript Name: SYS packages install
+# Description: |
+#   Installs packages required by RightScripts.
+#   To handle naming variations, prefix package names with "yum:" or "apt:".
+# Inputs:
+#   PACKAGES_INSTALL:
+#     Input Type: single
+#     Category: System
+#     Description: |
+#       Set to "false" in order to skip combined package installation of RS_PACKAGES.
+#     Required: true
+#     Advanced: true
+#     Default: text:true
+#     Possible Values:
+#       - text:true
+#       - text:false
+# ...
+
+[ "$PACKAGES_INSTALL" == "false" ] && exit 0
+
+if which apt-get > /dev/null 2>&1
```

Comment:
Nitpick and to follow a consistant pattern of the other scripts in this repo, pull the `then` on the same line as `if`:
`if which apt-get > /dev/null 2>&1; then`